tests/testthat/test-estimate_KM.R

```diff
@@ -43,6 +43,9 @@
 #' T7.2 The function prefixes the function call with survival
 #' T8. Piped datasets still return accurate results
 #' T8.1 Piped datasets still return accurate results
+#' T9. The user can specify formula argument
```

Comment:
I saw more things you implemented as safety nets and requirements eg when the string is empty you put the formula to NULL.

Reply:
- You will have to review above testing to be more specific eg we say "The function relies on the presence of two numeric variables, specified through `AVAL` and `CNSR`, to be present in `data`" but this is now only true when the formula is NULL.
- how are we testing the presence of the actual variables inside the formula argument? Are the NAs also removed from those variables, ...

Reply:
> eg when the string is empty you put the formula to NULL.

i am not sure what you're speaking to, this is not done

Reply:
- You will have to review above testing to be more specific eg we say "The function relies on the presence of two numeric variables, specified through `AVAL` and `CNSR`, to be present in `data`" but this is now only true when the formula is NULL.

formula is marked experimental...not need to update these in case implementation is changed

- how are we testing the presence of the actual variables inside the formula argument? Are the NAs also removed from those variables, ...

check and test added